src/Services/OrganizationsService.js

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+export const updateOrganization = (organizationId, fieldsToUpdate) => {
+  return axios
```

Comment:
@Jess-White I found this library that might be of use for removing some of the repetition we have when calling the rest api: https://resthooks.io/docs/getting-started/usage